packages/core/src/Stores/common-store.js

```diff
@@ -71,6 +71,11 @@ export default class CommonStore extends BaseStore {
         }
     }
 
+    @computed
+    get is_navigated_from_deriv_go() {
```

Comment:
```suggestion
    get if_from_mobile_app() {
```
need to change the name since we also have dp2p mobile app